Update worker-monitoring.adoc (#65)
mulesoft/docs-runtime-manager: modules/ROOT/pages/worker-monitoring.adoc

If you are setting this from the *Manage Application* menu, click *Apply Changes*.

> For applications running on runtime 3.x.y, upon deployment, CloudHub inserts a monitoring ping flow into your application. When the application starts, the logs display a message to indicate that monitoring is enabled and that the ping flow has been injected into your application.

Reviewer: For applications running on Mule 3.x.x...

Reviewer 2: ```suggestion
For applications running on runtime 3.8.x and below, upon deployment, CloudHub inserts a monitoring ping flow into your application. When the application starts, the logs display a message to indicate that monitoring is enabled and that the ping flow has been injected into your application.
```

Reviewer: Don't use "below." 3.8.x and earlier (less ambiguous)

> For applications running on runtime 3.8.x and below, upon deployment, CloudHub inserts a monitoring ping flow into your application. When the application starts, the logs display a message to indicate that monitoring is enabled and that the ping flow has been injected into your application.

Reviewer: 3.8.x and earlier (below is ambiguous)

== How Monitoring Works

> *Runtime 3.x.y* 

Reviewer: Mule runtime engine 3.x.x

Reviewer 2: ```suggestion
*Mule Runtime 3.8.x and below* 
```

> 

Reviewer: Every 30 seconds, CloudHub sends an HTTP GET request...

Logs should be lowercase--logs

> 

Reviewer: 3.8.x and earlier

Every thirty seconds, CloudHub sends an HTTP get request to the ping flow within the application. If the application is running normally, the ping flow sends a response. If CloudHub receives no response from the ping three times in a row, the Logs print a warning and display a full thread dump.

image::ch-monitoring-threaddump.png[CH-monitoring-threaddump]

> *Runtime 4.x.y*

Reviewer: Mule runtime engine 4.x.x

Reviewer 2: ```suggestion
*Mule Runtime 3.9.x and above*
```

> Every thirty seconds, CloudHub checks the PID of the application. If CloudHub detects the application process is not running three times in a row, the Logs print a warning.

Reviewer: Logs should be lowercase--logs

> Every thirty seconds, CloudHub checks the PID of the application. If CloudHub detects the application process is not running three times in a row, the Logs print a warning.

Reviewer 2: Should we write PID of the Mule Process instead?

> 

Reviewer: I'm not sure what "any" application means.  This seems to be talking about a particular application (ie CloudHub checks the PID of "the" application). If we are talking about a particular app, then it should be as it was before..."this" application. If any means something else like a worker for all applications stopped responding, then let's use a different word.

Author: We are talking about a particular app but trying to say that "CloudHub triggers a notification to the console when application stopped responding" applies to both 3.x.x and 4.x.x apps, no matter if they are checking ping flow or PID

Reviewer: Can we say something like:
For Mule 3.x.x and 4.x.x apps, CloudHub triggers a notification...
?

Author: Or can we say: In both cases, CloudHub triggers a notification... ?

Reviewer: @MichelleZMY That sounds good I think.

> *Mule Runtime 3.9.x and above*

Reviewer: 3.9.x and later
